packages/components/src/components/hds/advanced-table/helpers.ts

```diff
@@ -3,10 +3,7 @@
  * SPDX-License-Identifier: MPL-2.0
  */
 
-import { focusable, tabbable } from 'tabbable';
-
-import type { HdsAdvancedTableTdSignature } from './td.ts';
-import type { HdsAdvancedTableThSignature } from './th.ts';
+import { tabbable } from 'tabbable';
 
 export const onFocusTrapDeactivate = (cell: HTMLDivElement) => {
```

Comment:
question: should this helper be moved to `dom-management` too? it does operate on dom notes (changing the  `tabindex` attribute) as the other functions do

Reply:
I went back and forth on moving this function haha. You're right, its totally fair to just move it all to the dom management file because it does modify the dom.

Reply:
Updated.